stats/lib/platform-stats-fetchers.js

```diff
@@ -77,6 +77,40 @@ export const fetchDailyRewardTransfers = async (pgPool, filter) => {
   return rows
 }
 
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchAccumulativeDailyParticipantCount = async (pgPool, filter) => {
+  const { rows } = await pgPool.query(`
+    WITH daily_participants_count AS (
+      SELECT
+        day,
+        participant_id,
+        ROW_NUMBER() OVER (PARTITION BY participant_id ORDER BY day) AS rn
+      FROM daily_participants
+    ),
+    first_appearance AS (
+      SELECT day, participant_id
+      FROM daily_participants_count
+      WHERE rn = 1
+    ),
```

Comment:
It may be possible to simplify this first query as follows:

```sql
    WITH first_appearance AS (
      SELECT participant_id, MIN(day) as first_seen
      FROM daily_participants
      GROUP BY participant_id
    )
```

See https://github.com/filecoin-station/spark-stats/pull/232/files#diff-9c80f8ed9f9a7b436c7e194a16930630ea74a6af1c61484ea32e0914fe0a0c1fR177